core/src/main/java/jenkins/monitor/JavaVersionRecommendationAdminMonitor.java

```diff
@@ -78,15 +78,11 @@ public class JavaVersionRecommendationAdminMonitor extends AdministrativeMonitor
 
     static {
         NavigableMap<Integer, LocalDate> supportedVersions = new TreeMap<>();
-        // Adjust Java 11 end of life date for weekly and LTS
+        // Adjust Java 11 end of life date for LTS
         if (Jenkins.VERSION.split("[.]").length > 2) {
             // LTS will require Java 17 or newer beginning 30 Oct 2024
             // https://groups.google.com/g/jenkinsci-dev/c/gsXAqOQQEPc/m/VT9IBYdmAQAJ
             supportedVersions.put(11, LocalDate.of(2024, 9, 30)); // Temurin: 2024-10-31
```

Comment:
Seems this could be removed, as whatever upcoming LTS baseline includes this change will not work with Java 11 anyway, so this is effectively dead code.

Reply:
I don't have the brain cells left to think about this, so I will trust that your judgment is correct.